### PR TITLE
Add Guardian adapter file

### PR DESCRIPTION
**Pull request by grekuna**

### CODE REVIEW

File: lib/ecco/adapters/guardian.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+module Ecco
+  class GuardianAdapter < Adapter
+    attr_reader :page_block
+
+    def initialize
+      @root_uri = 'https://www.theguardian.com/international'
+      @article_block =
+      get_html(root_uri).css('.fc-item__content')
```
Comment (therod) [inline lib/ecco/adapters/guardian.rb]: Can you put this on the same line as the `@article_block` assignment?

File: lib/ecco/adapters/guardian.rb [hunk 2]
```diff
@@ -0,0 +1,21 @@
+module Ecco
+  class GuardianAdapter < Adapter
+    attr_reader :page_block
```
Comment (therod) [inline lib/ecco/adapters/guardian.rb]: Is there a specific reason why we need this additional attribute?

File: lib/ecco/adapters/guardian.rb (hunk 1 above)
Comment (grekuna) [inline lib/ecco/adapters/guardian.rb]: check

File: lib/ecco/adapters/guardian.rb (hunk 2 above)
Comment (grekuna) [inline lib/ecco/adapters/guardian.rb]: no, simplified the code in commit "refactor 1" to only include one attribute

**Comment by therod** [comment]
Perfect, thanks @grekuna !

**Pull request merged**